finans/datalonimport.php

```
@@ -102,7 +106,7 @@ function vis_data($kladde_id, $filnavn, $bilag){
 						while(!is_numeric(substr($linje,0,1)) && $linje) $linje=substr($linje,1);	
 					}
 				}
-				if ($preSetNo) list($dato[$y],$bilag,$faktura[$y],$beskrivelse[$y],$belob[$y],$tmp,$kontonr[$y])=explode(";",mb_convert_encoding($linje), 'UTF-8', 'ISO-8859-1');
+				if ($preSetNo) list($dato[$y],$bilag,$faktura[$y],$beskrivelse[$y],$belob[$y],$tmp,$kontonr[$y])=explode(";",mb_convert_encoding($linje, 'UTF-8', 'ISO-8859-1'));
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$tmp'. (undefined)

(UnusedLocalVariable)